docs/modules/record-encryption/proc-delete-lost-records.adoc

```diff
@@ -0,0 +1,54 @@
+// file included in the following:
+//
+// record-encryption-guide/con-lost-kek.adoc
+
+[id='proc-delete-lost-records-{context}']
+= Deleting Lost Records
+
+[role="_abstract"]
+This section describes how to delete records from a Kafka topic that are unrecoverable owing to a lost KEK.
+
+WARNING: Delete records from a topic is an irreversible option.
+
+.Prerequisites
+
+* Name of the topic affected.
+* Failing offset (this is reported in the error message reported by the client).
```

Comment:
This is wrong.  We need precise information about the topic/partition/offset that where the decrypt failure occurs.  We are currently not exposing that.